tools/setup_helpers/cudnn.py

```diff
@@ -1,97 +0,0 @@
-import os
-import glob
-
-from .env import IS_WINDOWS, IS_CONDA, CONDA_DIR, check_negative_env_flag, gather_paths, lib_paths_from_base
-from .cuda import USE_CUDA, CUDA_HOME
-
-
-USE_CUDNN = False
-CUDNN_LIB_DIR = None
-CUDNN_INCLUDE_DIR = None
-CUDNN_LIBRARY = None
-WITH_STATIC_CUDNN = os.getenv("USE_STATIC_CUDNN")
-
-if USE_CUDA and not check_negative_env_flag('USE_CUDNN'):
-    lib_paths = list(filter(bool, [
-        os.getenv('CUDNN_LIB_DIR')
-    ] + lib_paths_from_base(CUDA_HOME) + [
-        '/usr/lib/x86_64-linux-gnu/',
-        '/usr/lib/powerpc64le-linux-gnu/',
-        '/usr/lib/aarch64-linux-gnu/',
-        '/usr/lib/',
```

Comment:
CMake is able to detect architecture and therefore finds the correct dir without the need to specifying them explicitly